Milan   soft delete / migrations / jsonvalidator

MilanVHR opened the pull request.

MilanVHR committed:
Add softdelete to item_groups, clients, invenoties. Still needs unittests

MilanVHR committed:
Merge remote-tracking branch 'origin/development' into Milan---soft-delete

MilanVHR committed:
Tabjes :)

MilanVHR committed:
Add "required" Acces modifier in inventories- client- and ItemGroup models. SO you can't not give bodies with it

MilanVHR committed:
TOCH NIET REQUIRED BENUTTEN, GEEFT FOUTEN IN UNITTESTS :(

MilanVHR committed:
ADD [jsonRequired] to models, instead of requierd

MilanVHR committed:
ADD CETDatetime.now() instead of DateTime.now

MilanVHR committed:
Merge fout opgelost, oopsie

Ivan1084607 reviewed (changes requested):
verander je id naar
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int Id { get; set; }
    zodat het autogenerated wordt

Ivan1084607 reviewed (changes requested):
![image](https://github.com/user-attachments/assets/b864050b-7e9d-427e-a55b-b1589c54132e)
Email adress heeft geen [jsonRequrd]

MilanVHR committed:
change id to
[Key]
[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
public int Id { get; set; }

MilanVHR committed:
Add jsonrequired to email
